protocols/University_of_Albany-SUNY/PCR_Prep.ot2.py

```diff
@@ -0,0 +1,45 @@
+from opentrons import instruments, labware
+
+plate_96 = labware.load('96-flat', 1)
+
+plate_384 = labware.load('384-plate', 4)
+
+tiprack_10ul = labware.load('tiprack-10ul', 5)
+tiprack_300ul = labware.load('opentrons-tiprack-300ul', 2)
+
+tuberack = labware.load('tube-rack-2ml', 3)
+
+m10 = instruments.P10_Multi(
+    mount='right',
+    tip_racks=[tiprack_10ul]
+)
+
+s50 = instruments.P50_Single(
+    mount='left',
+    tip_racks=[tiprack_300ul]
+)
+
+for i in range(0, 2):
+    m10.distribute(3, plate_96(i), plate_384.rows(i))
```

Comment:
You should use brackets to index into the plates. Or you can do `plate_96.wells(i)` etc..